src/world.rs

```diff
@@ -341,7 +341,7 @@ impl World {
 
     /// Despawn all entities
     ///
-    /// Preserves allocated storage for reuse.
+    /// Preserves allocated storage for reuse but clears metadata so that [entity ID][Entity] will repeat (in contrast to [`despawn`][Self::despawn]).
```

Comment:
```suggestion
    /// Preserves allocated storage for reuse but clears metadata so that [`Entity`][Entity] values will repeat (in contrast to [`despawn`][Self::despawn]).
```